packages/vast-xml2js/src/helpers/xmlSelectors.js

```diff
@@ -3,31 +3,71 @@ const getChildren = ({elements = []} = {}) => elements;
 const findChildByName = (element, childName) =>
   getChildren(element).find(({name = ''}) => name.toUpperCase() === childName.toUpperCase());
 
-const filterChildrendByName = (element, childrentName) =>
-  getChildren(element).filter(({name = ''}) => name.toUpperCase() === childrentName.toUpperCase());
+const filterChildrenByName = (element, childrenName) =>
+  getChildren(element).filter(({name = ''}) => name.toUpperCase() === childrenName.toUpperCase());
 
+/**
+ * Get the first child element from the passed parsed xml element.
+ *
+ * @memberof module:@mol/vast-xml2js
+ * @param {Object} element - Parsed xml element object.
+ * @param {string} childName - Child element name
+ * @returns {Object|undefined} - the first child element with the passed name or undefined if not found.
+ */
 export const get = findChildByName;
 
+/**
+ * Get all the children elements of the passed parsed xml element filtered by the passed child name if passed.
+ *
+ * @memberof module:@mol/vast-xml2js
+ * @param {Object} element - Parsed xml element object.
+ * @param {string} [childName] - Child element name.
+ * @returns {Array} - Array of child elements or an empty array.
+ */
 export const getAll = (element, childName) => {
   if (typeof childName === 'string') {
-    return filterChildrendByName(element, childName);
+    return filterChildrenByName(element, childName);
   }
 
   return getChildren(element);
 };
 
-export const getFirstChild = (ad) => getChildren(ad)[0] || null;
+/**
+ * Get the first child element from the passed parsed xml element.
+ *
+ * @memberof module:@mol/vast-xml2js
+ * @returns {Object|null} - the first child element or undefined if there are none.
+ */
+export const getFirstChild = (element) => getChildren(element)[0] || null;
 
+/**
+ * Get the text value of the passed parsed xml element or null if there is non.
+ *
+ * @memberof module:@mol/vast-xml2js
+ * @returns {string|null} - text of the element or null.
+ */
 export const getText = (element) => {
   const firstChild = element && getFirstChild(element);
```

Comment:
I don't think you need to check if the element exist since:
```javascript
const getChildren = ({elements = []} = {}) => elements;
const getFirstChild = (element) => getChildren(element)[0] || null;
```